README.md

```diff
@@ -32,12 +32,30 @@ npm start
 
 Open http://localhost:8080 in any browser.
 
+## Mock mode (default)
+The app will run by default in **mock mode**, that means that all service implementation will be replaced by one mocked one with some fake data. This is useful for development, however it's also useful to run it with the actual implementation:
```

Comment:
**Note**: For some reason this was not included in my review... commenting again...

`replaced by one mocked one` => `replaced by a mocked one`